src/main/cpp/include/config/genomicsdb_config_base.h

```diff
@@ -161,6 +161,9 @@ class GenomicsDBConfigBase {
   inline bool enable_shared_posixfs_optimizations() const {
     return is_set_with_env_override(m_enable_shared_posixfs_optimizations, ENABLE_SHARED_POSIXFS_OPTIMIZATIONS);
   }
+  inline bool produce_lowercase_alleles_in_soft_masked_regions() const {
+    return m_produce_lowercase_alleles_in_soft_masked_regions;
+  }
```

Comment:
Does this need to be a configuration variable? I'm ok with simply trusting the input/reference genome

Reply:
Yeah I wasn't sure if we wanted the config knob either...from the sounds of it, GATK just capitalizes the bases. 

When we asked folks about whether any analyses might care about the distinction that the base was soft masked per the reference we just got a 🤷‍♂️ 

I'm fine with removing the config knob -- but I think our behavior should be to capitalize the base to stay consistent with GATK.

Reply:
Ah ok makes sense